docs/advisories/cve_2017_5715.md

```diff
@@ -0,0 +1,78 @@
+## CVE-2017-5715
+
+| NAME         	| CVE-2017-5715                                                                                                                                                                                          	|
+|--------------	|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|
+| Description  	| Systems with microprocessors utilizing speculative execution and branch prediction may allow unauthorized disclosure of information to an attacker with local user access via a side-channel analysis. 	|
+| Source       	| https://nvd.nist.gov/vuln/detail/CVE-2017-5715                                                                                                              |
+| NVD Severity 	| medium (attack range: local) |
+| Last Updated  | Jan 01 2018 |
+
+##  Solutions
```

Comment:
Add how to know:

```
  dmesg -H | grep 'page tables isolation'
    [  +0.000000] Kernel/User page tables isolation: enabled
```


If you _don't_ see "Kernel/User page tables isolation: enabled", you are vulnerable.